linker.ld:8: syntax error

[3/3] ld -o build/boot/boot.bin build/obj/kernel.o build/obj/loader.o -T ./src/linker.ld
FAILED: build/boot/boot.bin
ld -o build/boot/boot.bin build/obj/kernel.o build/obj/loader.o -T ./src/linker.ld
ld:./src/linker.ld:8: syntax error
ninja: build stopped: subcommand failed.

--- a/build.ninja
+++ b/build.ninja
@@ -1,4 +1,4 @@
-GPPPARAMS = -m32 -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-rtti -fno-exceptions -fno-leading-underscore -Wno-write-strings
+GPPPARAMS = -m32 -fno-use-cxa-atexit -nostdlib -fno-builtin -fno-rtti -fno-exceptions -fno-leading-underscore
 
 rule compile-cpp
  command = gcc $GPPPARAMS -o $out -c $in
@@ -7,18 +7,10 @@ rule compile-asm
  command = as --32 -o $out -c $in
 
 rule link
- command = ld -o $out $in -T $linkScript
-
+ command = ld -o fgos.exe ./src/kernel.o ./src/loader.o -T ./src/linker.ld
+	
 rule kernel
  command = sudo cp $in /boot/kernel.bin 
 
-# Path Variables
-bld  = ./build
-src  = ./src
-obj  = $bld/obj
-boot = $bld/boot
-
-build $obj/kernel.o:   compile-cpp $src/kernel.cpp
-build $obj/loader.o:   compile-asm $src/loader.s
-build $boot/boot.bin:  link        $obj/kernel.o $obj/loader.o
- linkScript = $src/linker.ld
+build ./src/kernel.o:  compile-cpp ./src/kernel.cpp
+build ./src/loader.o:  compile-asm ./src/loader.s

--- a/src/kernel.cpp
+++ b/src/kernel.cpp
@@ -5,7 +5,7 @@ void printf (char* str) {
     }
 }
 
-extern "C" void fgosmain () {
+extern "C" void fgosMain () {
     printf("Hello World");
     while(1);
 }

